go/ephemeral/selfprovision_test.go

```diff
@@ -0,0 +1,73 @@
+package ephemeral
+
+import (
+	"context"
+	"testing"
+
+	"github.com/keybase/client/go/engine"
+	"github.com/keybase/client/go/kbtest"
+	"github.com/keybase/client/go/libkb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEphemeralSelfProvision(t *testing.T) {
+	tc, user := ephemeralKeyTestSetup(t)
+	defer tc.Cleanup()
+
+	ctx := context.TODO()
+	g := tc.G
+	m := libkb.NewMetaContextForTest(tc)
```

Comment:
Super minor, but i'd probably do `ctx := m.Ctx()` here to keep it all in one context. On the other hand none of my tests did that, but I feel inspired now